overlord/devicestate/handlers_install.go

```diff
@@ -89,6 +90,28 @@ func writeModel(model *asserts.Model, where string) error {
 	return asserts.NewEncoder(f).Encode(model)
 }
 
+func writeLogs(rootdir string) error {
+	// XXX: would be great to use native journal format but it's tied
+	//      to machine-id, we could journal -o export but there
+	//      is no systemd-journal-remote on core{,18,20}
+	//
+	// XXX: or only log if persistant journal is enabled?
+	logPath := filepath.Join(rootdir, "var/log/install-mode.log")
```

Comment:
small suggestion to store this on the FAT partition appropriate for the device. We could also zip-compress it with a bunch of other files, like seed.yaml. This would be excellent for easy (windows) diagnostic of failed startup.

Reply:
That's an interessting idea. I wonder if there is any risk in storing this on a unencrypted partition, i.e. if it could contain sensitive information. This is why I opted for ubuntu-data but maybe this is more a development helper?

Reply:
I think the log could contain sensitive data so I think it should go on ubuntu-data

Reply:
how long does this take? do we know how big those logs are for example for our default amd64 install?

Reply:
The log is 214K on my nested-google amd64 install with debug. I have no exact timing data from the install bug running the equivalent command `journalctl > file` takes: `real	0m0.049s`.

Reply:
I agree that for now storing it on ubuntu-data is the safest approach.